apps/patient/src/views/Status.tsx

```diff
@@ -101,14 +94,21 @@ export const Status = () => {
     }
   }, [order?.fulfillment]);
 
-  if (error) {
-    return (
-      <Alert status="error">
-        <AlertIcon />
-        {error}
-      </Alert>
-    );
-  }
+  const initializePharmacy = async (p: EnrichedPharmacy) => {
+    await addRatingsAndHours(p);
```

Comment:
shouldn't there be a returned value? or is this mutating the object? this would be a side effect we should stay away from

wouldn't the function parameter be  `Pharmacy` and the return valued `EnrichedPharmacy`? Looks like we're just renaming the type

Reply:
yeah good call, i'll fix that